### ISSUE TITLE
Breadcrumb links on firefox/feature pages are misaligned

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14081]
## Description

The Protocol breadcrumbs component on the `firefox/feature` pages is misaligned with the rest of the page content.

### Steps to reproduce

Open https://www.mozilla.org/en-US/firefox/features/private/

### Expected result

The breadcrumb links should be aligned with the nav links

![image](https://github.com/mozilla/bedrock/assets/400117/d3242cbb-651f-423d-911f-152c526abe8b)

### Actual result

The breadcrumb is not aligned

![image](https://github.com/mozilla/bedrock/assets/400117/cde5fe17-de12-4753-aaed-8c2604843a3f)

It could also perhaps do with a little more top / bottom padding, to make the breadcrumb links easier to click?

### Environment

Firefox Nightly
macOS

**Comment by lucasferrazlima** [comment]
Hey @alexgibson!

I was looking into this and noticed that a style for the class `mzp-c-breadcrumb` already exists and is used for the published stories, such as in https://www.mozilla.org/en-US/stories/dreaming-then-building/

So I think a good solution, in order to be consistent, would be to include the same CSS bundles to `firefox/feature` pages. The breadcrumb component would then look as follows:

![image](https://github.com/mozilla/bedrock/assets/98855058/83ec4eba-cc65-4e5b-b0ba-8ffad7c6de8e)

I will proceed with a PR if you agree to this solution!

**Comment by reemhamz** [comment]
Hey @alexgibson, I took a closer look at this as I was reviewing @janbrasna's PR, and I think I came to understand that the reason there's considered to be a misalignment is actually because the Breadcrumb component is aligned similarly to the subnav that we have.

You can see this being played out in [VPN Resource Center articles](https://www.mozilla.org/en-US/products/vpn/resource-center/5-reasons-you-should-use-a-vpn/) (which has both a subnav and Breadcrumb component).

If you visit other pages around bedrock with subnavs, they all also have the same alignment.

Now the question is, do we want to:
1. Change the alignments of both subnav and Breadcrumb component to align better with the main nav of the site
2. Only change the Breadcrumb component alignment
3. Keep things as is

**Comment by janbrasna** [comment]
RIght, the alignment with subnav (e.g. in VPN resources) has been brought up too: #13462 — I guess people just expect the components to align somewhat more reasonably, probably within the bounds like this:

![Screenshot 2024-04-15 at 10 56 45](https://github.com/mozilla/bedrock/assets/1784648/6e5aeb2d-8baa-4589-b9da-ff6d1429e934)

👇 👇 👇 

![Screenshot 2024-04-15 at 10 56 45](https://github.com/mozilla/bedrock/assets/1784648/9830b7de-5f3f-49a5-b625-9ddd611fa9c8)

Would this be better addressed upstream, in Protocol, as a whole together with the subnav?

**Comment by alexgibson** [comment]
@janbrasna those proposed alignments make sense to me. I think we're OK to fix this in bedrock since the sub navigation component has yet to be back ported to Protocol officially (see https://github.com/mozilla/protocol/issues/921)

**Comment by stephaniehobson** [comment]
Closing in favour of https://github.com/mozilla/bedrock/issues/13462.

Hopefully a quick fix since https://github.com/mozilla/protocol/pull/1075 landed.
